wpe/src/main/cpp/browser/page.h

```diff
@@ -26,6 +26,8 @@ class Page
     Page(const Page&) = delete;
     Page& operator=(const Page&) = delete;
 
+    virtual ~Page();
```

Comment:
If you don't have a virtual destructor, you should set the class `final` to avoid any misuse with polymorphism.

Indeed, I would have kept the destructor because in case that Java code calls pageDestroy() without calling pageClose() first, you ensure that resources are correctly freed.

It is always a good idea to ensure allocation/desallocation of resources at the same place and not rely exclusively on external calls.

Reply:
Maybe you forgot to push a commit because here it appears fixed but in the PR code it is not.

Reply:
Page is marked as final now